common/network-common/src/main/java/org/apache/spark/network/client/TransportClientFactory.java

```diff
@@ -172,10 +172,13 @@ public TransportClient createClient(String remoteHost, int remotePort)
     final long preResolveHost = System.nanoTime();
     final InetSocketAddress resolvedAddress = new InetSocketAddress(remoteHost, remotePort);
     final long hostResolveTimeMs = (System.nanoTime() - preResolveHost) / 1000000;
+    final String resolvMsg = resolvedAddress.isUnresolved() ? "failed" : "succeed";
     if (hostResolveTimeMs > 2000) {
-      logger.warn("DNS resolution for {} took {} ms", resolvedAddress, hostResolveTimeMs);
+      logger.warn("DNS resolution {} for {} took {} ms",
+              resolvMsg, resolvedAddress, hostResolveTimeMs);
```

Comment:
The indent is too deep; can it just do on the previous line? if not continuation indent is at most 4 spaces

Reply:
Hi @srowen 
I thought the line length was limited too. Pushed the fix.